### PR TITLE
Try This

### PR DESCRIPTION
**Pull request by AntsInMyEyesJohn**
Just because

### CODE REVIEW

File: newCode [hunk 1]
```diff
@@ -1,6 +1,6 @@
-int x = 8;
-iny y = 9;
+int x = 6;
+iny y = 5;
 
-answer = x * y; 
+answer = x + y; 
 
 Console.WriteLine(answer);
```
Comment (joryhouse) [inline newCode:6]: This should print to the console the answer

File: newCode [hunk 2]
```diff
@@ -1,6 +1,6 @@
-int x = 8;
-iny y = 9;
+int x = 6;
+iny y = 5;
 
-answer = x * y;
```
Comment (joryhouse) [inline newCode:4]: This is declaring answer

File: newCode (hunk 1 above)
Comment (AntsInMyEyesJohn) [reply newCode:6]: Ok

File: newCode (hunk 2 above)
Comment (AntsInMyEyesJohn) [reply newCode:4]: Ok

**Pull request merged**